pkg/cli/mirror/mirror.go

```diff
@@ -169,6 +170,32 @@ func (o *MirrorOptions) Complete(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+// checkDockerReference prints warnings or returns an error if applicable.
+func checkDockerReference(mirror imagesource.TypedImageReference) error {
+	switch {
+	case mirror.Ref.Registry == "" && mirror.Ref.Namespace != "" && strings.Count(mirror.Ref.Name, "/") >= 1:
+		klog.Warningln("The docker reference was parsed as a namespace and a repository name, not including a registry.")
+		klog.Warningln("To specify a registry, use a qualified hostname.")
```

Comment:
@d10n `klog.Warning` logs to twice. Warning and info. Maybe change to another level or use `fmt.Fprintf`?

This is what I am seeing:
```
The docker reference was parsed as a repository (or image) name, not a registry.
The docker reference was parsed as a repository (or image) name, not a registry.
To specify a repository, use a qualified hostname.
To specify a repository, use a qualified hostname.
For example, instead of docker://registry, use docker://registry.localdomain
For example, instead of docker://registry, use docker://registry.localdomain
```